OpenKeychain/src/main/java/org/sufficientlysecure/keychain/provider/KeychainDatabase.java

```diff
@@ -54,7 +54,7 @@
  */
 public class KeychainDatabase extends SQLiteOpenHelper {
     private static final String DATABASE_NAME = "openkeychain.db";
-    private static final int DATABASE_VERSION = 16;
+    private static final int DATABASE_VERSION = 17;
```

Comment:
The db version is increased to trigger a ConsolidateOperation once. The uid parsing gives different results in some cases now, but the user_packets.[name, email, comment] fields in the db do not reflect that. In most cases this is not problematic, as only the uid is taken and simply parsed again, but e.g. for the change in KeychainProvider introduced in this PR it is.